tests/test_extensions.py

```diff
@@ -0,0 +1,132 @@
+"""
+Unit tests for Extension Manager
+"""
+
+import sys
+import os
+import unittest
+import mock
+import shutil
+
+dir_of_executable = os.path.dirname(__file__)
```

Comment:
> Even if you didn't like tox you would run the tests with nosetest or pytest because you don't want to run a single test file.

If I implement a feature on `ExtensionManager`, I don't want to run other tests. Currently, all of our tests are pretty quick, but in the future, they make take a number of seconds to finish.